Azure-Samples/azure-ml-federated-learning
# Display metrics at the pipeline level

## Purpose

* Log training metadata in MLFlow to showcase metrics at the pipeline level.
* Report train/test loss and test accuracy for each round.
* Display metrics for each as well as combined rounds corresponding to all silos.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code


Submit a pipeline and then go to the AML UI to see metrics in the "Pipeline Job overview -> Metrics"
